iset.mm

```diff
@@ -136047,6 +136061,163 @@ S C_ ( P ( ball ` D ) T ) ) $=
   $}
 
 
+$(
+###############################################################################
+  BASIC REAL AND COMPLEX ANALYSIS
+###############################################################################
+$)
+
+
+$(
+#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
+  Derivatives
+#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
+$)
+
+
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Real and complex differentiation
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Limit operator
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  $c limCC $. $( Limit operator $)
+
+  $( The limit operator. $)
+  climc $a class limCC $.
+
+  ${
+    $d d e f x y z $.
+    $( Define the set of limits of a complex function at a point.  Under normal
+       circumstances, this will be a singleton or empty, depending on whether
+       the limit exists.  (Contributed by Mario Carneiro, 24-Dec-2016.) $)
+    df-limced $a |- limCC = ( f e. ( CC ^pm CC ) , x e. CC |->
```

Comment:
This definition is very different from the one in set.mm, therefore "(Revised by Jim Kingdon, 3-Jun-2023.) " should be added.

Reply:
Agreed, it is pretty much a different definition. Should it just be credited to me, or take the credit from ellimc3 (which I guess in this case is mostly the same thing since df-limc is from the same author in the same year), also with a Revised line?

Reply:
I think adding "(Revised by Jim Kingdon, 3-Jun-2023.)" would be sufficient.

Reply:
Done.